benchmark_results.md

```
@@ -0,0 +1,58 @@
+| Real Data Queries | 91810 | 18465ns | 5KB | 43 |
+| Block Nonce Lookup Real Data | 34676 | 38519ns | 4KB | 73 |
+| Pool Lookup By Key Hash Real Data | 39112 | 33676ns | 4KB | 79 |
+| D Rep Lookup By Key Hash No Data | 35720 | 33017ns | 4KB | 77 |
+| Transaction History Queries No Data | 35151 | 35753ns | 4KB | 78 |
+| Datum Lookup By Hash Real Data | 44521 | 32613ns | 4KB | 69 |
+| Block Nonce Lookup No Data | 31076 | 38415ns | 4KB | 73 |
+| Account Lookup By Stake Key No Data | 34257 | 33124ns | 4KB | 75 |
+| Transaction History Queries Real Data | 37896 | 39330ns | 4KB | 78 |
+| Concurrent Queries | 19658 | 58155ns | 172221queries/sec | 3851 |
+| D Rep Lookup By Key Hash Real Data | 39349 | 35230ns | 4KB | 77 |
+| Block Memory Usage | 25939 | 48725ns | 14KB | 49 |
```

markdownlint-cli2 (0.18.1)

Tables should be surrounded by blank lines

(MD058, blanks-around-tables)